packages/eas-cli/src/commands/secret/create.ts

```diff
@@ -33,7 +40,11 @@ export default class EnvironmentSecretCreate extends EasCommand {
       description: 'Name of the secret',
     }),
     value: Flags.string({
-      description: 'Value of the secret',
+      description: 'Text value or path to a file to store in the secret',
+    }),
+    type: Flags.enum({
+      description: 'Type of the secret (use to force the type)',
```

Comment:
This makes more sense grammatically to me. We could also update the descriptions above to "The secret's name", "The secret's value" to feel a bit more human

```suggestion
      description: 'The type of secret (string is the default type)',
```